property.go

```diff
@@ -106,6 +107,19 @@ func parseProperty(propData interface{}, matchName string) (*Property, error) {
 			if err == nil {
 				prop.Name = fnamBlock.ToString()
 			}
+			tdgpBlock, err := propHead.SublistFind("tdgp")
+			if err == nil {
+
+				// Look for a tdsn which specifies the user-defined label of the property
+				tdsnBlock, err := tdgpBlock.FindByType("tdsn")
+				if err == nil {
+					label := fmt.Sprintf("%s", tdsnBlock.ToString())
+
+					if label != "-_0_/-" {
```

Comment:
Does this label have any specific special meaning? If so, a comment explaining the significance of `-_0_/-` would be good to document here